packages/mui-lab/src/Masonry/Masonry.test.js

```diff
@@ -28,28 +28,38 @@ describe('<Masonry />', () => {
 
   describe('render', () => {
     it('should render with correct default styles', function test() {
-      if (!/jsdom/.test(window.navigator.userAgent)) {
+      if (/jsdom/.test(window.navigator.userAgent)) {
```

Comment:
Should probably reject `toHaveComputedStyle` in JSDOM as well. But I don't want to be overzealous so I think we can defer that restriction if it becomes a problem.
Remember: If you are confused with development tools or identify common blockers you experience, you have to share them with the team. These issues don't resolve themselves.